apps/playground-api/__tests__/modules/PlaygroundModule.test.ts

```diff
@@ -193,4 +201,24 @@ it('should have gov set', async () => {
     ]
   })
   await expect(promise).rejects.toThrow('Payback of loan via DUSD token is not currently active')
+
+  // future swap test
+  const swapAddr = await testing.generateAddress()
+  await testing.container.call('sendtokenstoaddress', [{}, { [swapAddr]: ['2@TR50'] }])
+  await testing.generate(1)
+
+  await waitForPriceValid()
+  await testing.rpc.account.futureSwap({
+    address: swapAddr,
+    amount: '2@TR50'
+  })
+  await testing.generate(1)
+
+  const pending = await testing.container.call('listpendingfutureswaps')
+  expect(pending.length).toStrictEqual(1)
+
+  const attributes = await testing.rpc.masternode.getGov('ATTRIBUTES')
+  expect(attributes.ATTRIBUTES['v0/params/dfip2203/active']).toStrictEqual('true')
+  expect(attributes.ATTRIBUTES['v0/params/dfip2203/block_period']).toStrictEqual('10')
+  expect(attributes.ATTRIBUTES['v0/params/dfip2203/reward_pct']).toStrictEqual('0.05')
 })
```

Comment:
Can generate 10 more blocks and check futureswaps settled right?

Reply:
yes.. gen(10) is for sure